src/app/components/jobs-slider/jobs-slider.component.scss

```diff
@@ -0,0 +1,109 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-jobs-slider-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 3em;
+  @media screen and (max-width: $md-device-width) {
+    padding: 3em 1em;
+  }
+
+  .home-jobs-slider-new-jobs-title {
+    color: $color-secondary-gray;
+  }
+
+  .home-jobs-slider-user-jobs-title {
+    color: $white;
+  }
+
+  .home-jobs-slider-show-all-jobs-link-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column-reverse;
+
+    .home-jobs-slider-show-all-jobs-link {
+      cursor: pointer;
```

Comment:
Properties should be ordered padding-left, cursor

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)